CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Adds
 
 * Adds support for `type` query parameter for page autocomplete. This allows to filter the results by page type. Example: `/api/v1/@apostrophecms/page?autocomplete=something&type=my-page-type`.
+* Adds support for browsing specific pages in a relationship field when `withType` is set to a page type, like `home-page` for example.
```

Comment:
`home-page` is not a great example since there is only one. I suggest `default-page`

Reply:
I thought `home-page` was better since it exists in the core, whereas `default-page` doesn't.
I'll add several examples to make it less confusing